attestation-service/src/token/ear_broker.rs

```diff
@@ -204,22 +203,14 @@ impl EarAttestationTokenBroker {
 impl AttestationTokenBroker for EarAttestationTokenBroker {
     async fn issue(
         &self,
-        tcb_claims: TeeEvidenceParsedClaim,
+        all_tee_claims: Vec<TeeClaims>,
         policy_ids: Vec<String>,
         init_data_claims: serde_json::Value,
         runtime_data_claims: serde_json::Value,
         reference_data_map: HashMap<String, Vec<String>>,
-        tee: Tee,
+        _tee: Tee,
```

Comment:
Ok I see this good design to deliver Vec<TeeClaims> to token broker here. `tee` parameter should be of no use but the left over here is to apply for sample token. I can put another PR to get rid of the `tee` parameter here.